source/puddletag

```diff
@@ -348,9 +348,11 @@ if __name__ == '__main__':
 
     # Check if dirnames passed on command line.
     if filenames:
-        dirname = filenames[0]
-        if dirname and os.path.exists(dirname):
-            win.openDir(dirname, False)
```

Comment:
It looks like you removed the functionality of specifying a directory?

Reply:
No, opening a directory still works. If you run "puddletag dirname" you end up with an array of filenames containing just ["dirname"], and that gets opened here:

    win.openFiles(filenames, False)

It loads all the files under that directory, same as before.